content/pt-br/docs/reference/glossary/toleration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Tolerância
+id: toleration
+date: 2019-01-11
+full_link: /pt-br/docs/concepts/scheduling-eviction/taint-and-toleration/
+short_description: >
+  Um objeto principal consistindo de três propriedades principais: chave, valor, e efeito. Tolerâncias habilitam o agendamento de pods em nós ou grupos de nós que tenham uma taint coincidente.
```

Comment:
"Pods" instead of "pods". Majority terms in the glossary use Pod capitalized. Check below too.

In this case I prefer use "alocar" instead of "agendar". Think about it "habilitam a alocação de Pods em nós...". If you agree, check text below.

Reply:
Hey @MrErlison :wave: 

Thanks for your input, i tried to change all occurrences of `agendar` by `alocar`.

Let me know if it looks better!